### PR TITLE
nsquare7.json

### PR DESCRIPTION
**Pull request by nsquare7**
this is my location
**here** is *nothing*.
*@githubteacher*
closes #5066

### CODE REVIEW

File: _pins/nsquare7_1.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle : nsquare7
+Latitude :27.6667
+Longitude: 85.3167
+---
```
Comment (a-a-ron) [inline _pins/nsquare7_1.json]: Hey @nsquare7, you're almost there. Make sure your format follows the below example exactly.

```
---
githubHandle: YOUR-USERNAME
latitude: YOUR_LATITUDE
longitude: YOUR_LONGITUDE
---
```
Make the `Latitude` and `Longitude` lowercase and your builds should pass.

Comment (hectorsector) [inline _pins/nsquare7_1.json]: @nsquare7 I also noticed your filename is `nsquare7_1.json` -- you can go ahead and just use `nsquare7.json` so your avatar is included in the final map for the activity!

**Comment by hollenberry** [comment]
Hey @nsquare7  ! Looks like you've gotten great advice from Hector and Aaron above! If you take action on those changes, you'll be set. 👍

Please let us know if you need any help! We'll be closing this out to keep the repository clean if there's no action or conversation within the week.

**Comment by crichID** [comment]
@nsquare7 I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/5066/commits/8d5810c9cc949c4a654761861ecc40b2c42e3a9f. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by April 21st, I will go ahead and do it just to keep the repository clean!

**Pull request merged**